src/components/sporsmal/Sporsmal.tsx

```diff
@@ -0,0 +1,266 @@
+        <>
+            <AlertStripeHjelper
+                vis={formState.isSubmitted && !formState.isValid}
+                type="feil"
+                tekst={tekster['alertstripe.feil-i-utfyllingen.tekst']}
+            />
+            <form onSubmit={handleSubmit(onSubmit)}>
+                <PanelBase className="panelbase">
+                    <SkjemaGruppe
+                        feil={
+                            errors.opplysningeneErRiktige
+                                ? { feilmelding: tekster['jaEllerNei.feilmelding'] }
+                                : undefined
+                        }
+                    >
+                        <Fieldset legend={tekster['jaEllerNei.tittel']}>
+                            <Radio
+                                label={tekster['ja']}
+                                name="opplysningeneErRiktige"
+                                value={JaEllerNei.JA}
+                                radioRef={register as any}
+                            />
+                            <Radio
+                                label={tekster['nei']}
+                                name="opplysningeneErRiktige"
+                                value={JaEllerNei.NEI}
+                                radioRef={register as any}
+                            />
+                        </Fieldset>
+                    </SkjemaGruppe>
+                    <OpplysningeneErFeil
+                        vis={watchOpplysningeneErRiktige === JaEllerNei.NEI}
+                        visAlertstripeAvbryt={watchPeriode || watchSykmeldingsgrad}
+                        visAlertstripeBrukArbeidsgiver={!(watchPeriode || watchSykmeldingsgrad) && watchArbeidsgiver}
+                        visAlertstripeBruk={
+                            !(watchPeriode || watchSykmeldingsgrad || watchArbeidsgiver) &&
+                            (watchDiagnose || watchAndreOpplysninger)
+                        }
+                        register={register}
+                        errors={errors}
+                    />
+                </PanelBase>
+                <Vis
+                    hvis={
+                        watchOpplysningeneErRiktige === JaEllerNei.JA ||
+                        (!watchPeriode &&
+                            !watchSykmeldingsgrad &&
+                            (watchArbeidsgiver || watchDiagnose || watchAndreOpplysninger))
+                    }
+                >
+                    <PanelBase className="panelbase">
+                        <SkjemaGruppe
+                            feil={errors.sykmeldtFra ? { feilmelding: tekster['sykmeldtFra.feilmelding'] } : undefined}
+                        >
+                            <Fieldset
+                                legend={
+                                    <div>
+                                        {tekster['sykmeldtFra.tittel']}
+                                        <Hjelpetekst>{tekster['sykmeldtFra.hjelpetekst']}</Hjelpetekst>
```

Comment:
Anbefaler å wrappe denne teksten med noe slikt for å begrense bredden:

`
                                        <Hjelpetekst>
                                            <div style={{ maxWidth: 300 }}>{tekster['sykmeldtFra.hjelpetekst']}</div>
                                        </Hjelpetekst>`